beacon-chain/sync/validate_beacon_blocks.go

```diff
@@ -84,6 +84,7 @@ func (s *Service) validateBeaconBlockPubSub(ctx context.Context, pid peer.ID, ms
 			blockHeader, err := interfaces.SignedBeaconBlockHeaderFromBlockInterface(blk)
 			if err != nil {
 				log.WithError(err).WithField("blockSlot", blk.Block().Slot()).Warn("Could not extract block header")
+				return
 			}
 			s.cfg.slasherBlockHeadersFeed.Send(blockHeader)
```

Comment:
Not sure about returning and skipping the rest of the validation. How about an else?

```suggestion
			} else {
			s.cfg.slasherBlockHeadersFeed.Send(blockHeader)
			}
```

Reply:
It's incapsulated in function call, so return seems fine here, it only skip sending empty block header to feed

Reply:
Missed that. You are right!